tests/Middleware/HttpCacheTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Tests\Middleware;
+
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Nyholm\Psr7\Response;
+use Nyholm\Psr7\ServerRequest;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Http\Method;
+use Yiisoft\Yii\Web\Middleware\HttpCache;
+
+class HttpCacheTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function validCacheResult(): void
```

Comment:
What's exactly tested here? 200 should be returned if content was modified since X along with the body. The X is current time. It's a bit confusing overall.